modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -81,9 +81,10 @@ public ObjectDefinition postObjectDefinition(
 
 		return _toObjectDefinition(
 			_objectDefinitionService.addCustomObjectDefinition(
-				LocalizedMapUtil.getLocalizedMap(objectDefinition.getName()),
+				LocalizedMapUtil.getLocalizedMap(objectDefinition.getLabel()),
```

Comment:
Hey @gabrielwas , these fields have already been created in the UI? I passed label and pluralLabel with name value to not break the UI while Adriano is working on that.

Reply:
@guilhermedcamacho I'm filling the labelMap if it is empty in the service layer.